lib/get-commits.js

```diff
@@ -18,7 +18,7 @@ module.exports = async ({cwd, env, lastRelease: {gitHead}, logger}) => {
 
   Object.assign(gitLogParser.fields, {hash: 'H', message: 'B', gitTags: 'd', committerDate: {key: 'ci', type: Date}});
   const commits = (await getStream.array(
-    gitLogParser.parse({_: `${gitHead ? gitHead + '..' : ''}HEAD`}, {cwd, env: {...process.env, ...env}})
+    gitLogParser.parse({ _: [`${gitHead ? gitHead + '..' : ''}HEAD`,  ...(commitPaths ? ['--', ...commitPaths] : '')]}, {cwd, env: {...process.env, ...env}})
```

Comment:
```suggestion
    gitLogParser.parse({ _: [`${gitHead ? gitHead + '..' : ''}HEAD`,  ...(commitPaths ? ['--', ...commitPaths] : [])]}, {cwd, env: {...process.env, ...env}})
```

Reply:
Yes, I will make the recommend change and look at working on the documentation updates tonight/tomorrow.